content/installation/upgrading-manager.md

```diff
@@ -2,96 +2,113 @@
 layout: bt_wiki
 title: Upgrading Cloudify Manager
 category: Installation
-draft: true
+draft: false
 weight: 350
 
 ---
 
-{{% gsTip title="Version Relevance" %}}
-You can use this process to upgrade an existing Cloudify Manager 4.x to a later version.
-{{% /gsTip %}}
+To get the latest features and benefits of Cloudify Manager, we recommend that you upgrade to the latest version of Cloudify. You can upgrade in one of these ways:
 
-* **Reinstall** - Replace Cloudify Manager with the same version on the same host
-* **Migration** - Replace Cloudify Manager with the same version on another host
 * **In-place upgrade** - Replace Cloudify Manager with a higher version on the same host
 * **Migration upgrade** - Replace Cloudify Manager with a higher version on another host
 
 {{% gsNote title="Version Relevance" %}}
-The upgrade process is supported for upgrade from any currently supported version. To upgrade from an unsupported version, contact Cloudify Support.
-{{% /gsNote %}}
+The upgrade process is supported for upgrade from any currently supported version to the latest GA version. To upgrade from an unsupported version, contact Cloudify Support. 
 
-{{% gsNote title="Caution" %}}
-Make sure that no users are connected to any manager services during the upgrade or migration.
+When you upgrade to Cloudify to 4.2 and above, you must also consider [user roles and permissions]({{< relref "installation/upgrading-manager.md#upgrading-into-the-new-roles-system" >}}).
 {{% /gsNote %}}
 
-The upgrade or migration process includes:
-
-* Reinstall -
-
-  1. Save snapshot of the Cloudify Manager.
-  1. Uninstall the Cloudify Manager from the host
-  1. Install the Cloudify Manager on the host.
-  1. Restore snapshot of the Cloudify Manager to the host.
-
-* Migration -
+{{% gsWarning title="Downtime" %}}
```

Comment:
We seem to be summarising the process in a different place again (see comment on migrations).